pkg/local_workflows/code_workflow/code_client_helper.go

```diff
@@ -25,3 +28,18 @@ func (c *codeClientConfig) SnykCodeApi() string {
 func (c *codeClientConfig) SnykApi() string {
 	return c.localConfiguration.GetString(configuration.API_URL)
 }
+
+func (c *codeClientConfig) SnykCodeAnalysisTimeout() time.Duration {
+	var snykCodeTimeout time.Duration
+	var err error
+	env := c.localConfiguration.GetString(configuration.TIMEOUT)
```

Comment:
Suggestion: User `GetInt()` instead of `getString()` and maybe even `isSet()` to keep the code simpler.

Reply:
Great suggestion, all done in https://github.com/snyk/go-application-framework/pull/208/commits/3ea752e21e26a465736cd32b55480c283a299e5f